### PR TITLE
added progress bar to event creation form

### LINKED ISSUE #135: Add a progress bar at the end of page having form
Some pages on vms have large number of fields. Google form has a progress bar in the end of the form. It is nice for the user to know how much is done. Show a progress bar at the end of such pages.

### PR DESCRIPTION
**Pull request by varunnaganathan**
closes #135 

Basically used the HTML progress element to display the progress bar for the event form.I used  javascript and jquery to update the progress bar when the user fills in data in each form field.
Screenshots:
progress1
progress2
progress3

### CODE REVIEW

**Comment by tapasweni-pathak** [comment]
Can you please add some screenshots?

**Comment by varunnaganathan** [comment]
@tapasweni-pathak I'm really sorry.Will take care of your suggestion in future.
The unnecessary commits was my bad.I thought my repo was in sync with the vms develop branch while it was not.Some merge conflicts were causing a problem in the fetching of vms upstream.
After clearing those,i was able to solve that problem.
As for the screen shots:
progress1
progress2
progress3

**Comment by tapasweni-pathak** [comment]
> I'm really sorry.Will take care of your suggestion in future.

No issues. It's okay. Great. :smile: 

> The unnecessary commits was my bad.I thought my repo was in sync with the vms develop  branch while it was not.Some merge conflicts were causing a problem in the fetching of vms upstream.
After clearing those,i was able to solve that problem.

Cool. :+1: 

>As for the screen shots.

Add these screenshots in the description of the PR. With this anyone who is reviewing your PR will get the complete idea about the changes as soon as they look at it.

**Comment by varunnaganathan** [comment]
Hi @tapasweni-pathak , No reviews yet. Let me know if any changes needed.Thanks

**Comment by tapasweni-pathak** [comment]
@varunnaganathan I'm occupied with something else. I'll take some time before I start reviewing all the pending pull request's. Hope you understand.

**Comment by varunnaganathan** [comment]
Ok @tapasweni-pathak .Thanks for the prompt reply.

File: vms/event/templates/event/create.html [hunk 1]
```diff
@@ -1,8 +1,90 @@
...
+
+$('#v8').change(function() {
+    valeur+=a8;
+    a8=0;
+     document.getElementById("myProgress").value = valeur;
+   
+   
+});
+});
+</script>
```
Comment (tapasweni-pathak) [inline vms/event/templates/event/create.html]: Please separate this JS file. See if this code can be merged in some already existing JS file or create a new one.

**Comment by tapasweni-pathak** [comment]
@varunnaganathan Can you please do the suggested changes?

**Comment by tapasweni-pathak** [comment]
Closing this. Please reopen if rebased or create a fresh pull request.

**Pull request closed without merging**